src/ToCode/Property.cs

```diff
@@ -70,6 +70,13 @@ public virtual void SetValue(object? value)
                     b.Width = s.Length + (b.IsDefault ? 6 : 4);
                 }
             }
+
+            // some views don't like null and only work with "" e.g. TextView
+            // see https://github.com/gui-cs/TerminalGuiDesigner/issues/91
+            if (value == null)
+            {
+                value = ustring.Make("");
+            }   
```

Comment:
Yes, perhaps there are many bugs on the views that will change it's behavior after I changed the `NStack.ustring.IsNullOrEmpty` method. If there are views that want invalidate for `null` and `empty` theses expressions will fail for the both cases `if (value == null)` or `if (value == "")`. If there are views that want invalidate only for `null` then this expression will fail `if (value == "")`. If there are views that want invalidate only for `empty` then this expression will fail `if (value == null)`. I already changed something in the `TextView` and `TextField` when I fixed the `Clipboard` tests, but I'm certain there are more cases. Your workaround will help in the situations where the views are validate only for empty, but helps a lot. Later the solution is fixing this issues in the views as they come up.